cni-plugin/pkg/plugin/plugin.go

```diff
@@ -185,6 +214,31 @@ func cmdAdd(args *skel.CmdArgs) (err error) {
 		return
 	}
 
+	if conf.WaitForReadyEndpoint {
+		if conf.ReadinessEndpoint != "" {
+			if _, err := url.ParseRequestURI(conf.ReadinessEndpoint); err != nil {
+				return fmt.Errorf("Invalid URL set for parameter ReadinessEndpoint:%s Error:%v",
+					conf.ReadinessEndpoint, err)
+			}
+		}
+
```

Comment:
I think moving the `if readyEndpoint == ""` logic here and passing pollInterval as an arg to `isEndpointReady()` would concentrate all uses of the "global" consts in one place, which can be good for wrapping our head around the code (at least that's how I would think, @caseydavenport feel free to shrug this off)

Reply:
Agree with reducing the number of globals - localized variables make the code much easier to read.

Reply:
If I understood correctly @coutinhop wanted to have all global variables used at one place. Still those variables are global, please let me know if it is not what you want.